src/shared/components/palette/case-link/domain/linked-cases.model.ts

```diff
@@ -32,3 +32,20 @@ export class LinkReason {
   reason: string;
   otherDescription?: string;
 }
+
+export class GetLinkedCases {
```

Comment:
The naming convention of the class name `GetLinkedCases` looks like a property or method name.

Reply:
The class properties missing semi-colon.

Reply:
agreed with John's comments